style.css

```diff
@@ -62,3 +62,489 @@ nav {
   line-height: 24px;
   background: #0e0e0f;
 }
+
+.text3 {
+  position: absolute;
+  width: 181px;
+  left: 16px;
+  top: 650px;
+  font-family: 'Playfair Display', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.projectpageimage1 {
+  background: url("Rectangle_21.svg");
+  display: flex;
+  width: 100%;
+  height: 186px;
+  top: 500px;
+  position: relative;
+  border: 0.333333px solid rgba(235, 86, 86, 0.1);
+}
+
+.projectpageimage2 {
+  background: url("second_project.svg");
+  display: flex;
+  width: 100%;
+  height: 186px;
+  top: 414px;
+  position: relative;
+  border: 0.333333px solid rgba(235, 86, 86, 0.1);
+}
+
+.projectpageimage3 {
+  background: url("thirdproject.svg");
```

Comment:
- [ ] Please check your background image is repeating in the document to try to fix this issue. 
Hint:- use of `no-repeat` property.